doc/doxyfile.in

```diff
@@ -44,14 +44,14 @@ PROJECT_NUMBER         = @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = 
+PROJECT_BRIEF          =
```

Comment:
Maybe someone should submit a patch for [this line](https://github.com/doxygen/doxygen/blob/master/addon/doxywizard/expert.cpp#L755) in doxywizard's source? :grin: